src/stream/src/executor/source/source_executor.rs

```diff
@@ -260,6 +262,9 @@ impl<S: StateStore> SourceExecutor<S> {
             }
         }
 
+        let mut max_message_per_sec = 1024;
+        let rate_limit_interval = Duration::from_millis(1000);
```

Comment:
Are these fair initial values?

Reply:
just placeholder, kexiang will look into it